### ISSUE TITLE
[v2] Connecting to a cluster should not try to create ambassador namespace if already present

### ISSUE THREAD

**Issue by maxlegault** [telepresenceio/telepresence#1678]
I am running in an issue with Telepresence 2.2.0 when trying to connect to a cluster in which an `ambassador` namespace exists, but the `traffic-manager` deployment and services have been removed (from a `telepresence uninstall --everything`). 

When running `telepresence connect` in that configuration, I get the following error:
```
error: namespaces is forbidden: User "<redacted>" cannot create resource "namespaces" in API group "" at the cluster scope: requires one of ["container.namespaces.create"] permission(s).
```

I am admin in the `ambassador` namespace, but not in the whole cluster. I should still be able to create the `traffic-manager` deployment and service when the namespace exists.

**Comment by donnyyung** [comment]
This was fixed in https://github.com/telepresenceio/telepresence/pull/1679 and will be in the next release of telepresence :)

**Comment by khussey** [comment]
This is fixed in the 2.2.1 release, which is now available.
